manage_rentals.rb

```diff
@@ -0,0 +1,35 @@
+require './rental'
+
+class ManageRentals
+  attr_accessor :rentals
+
+  def initialize
+    @rentals = []
+  end
+
+  def create_rental(book_list, person_list)
+    book_list.list_books_indexes
+    book = book_list.return_books_array[new_index(book_list.return_books_array)]
+
+    person_list.list_persons_indexes
+    person = person_list.return_persons_array[new_index(person_list.return_persons_array)]
+
+    date = new_string { 'Date: ' }
+
```

Comment:
- [ ] I am trying to add the date but it will be asking continuously after adding as well. Please fix this.

![Screenshot from 2022-04-12 23-16-55](https://user-images.githubusercontent.com/22434660/163023602-d96c7f25-db47-4605-91a7-46ac1c18b1c1.png)